### PR TITLE
Angular support doc

### PR DESCRIPTION
**Pull request by wunaidage**
##### Checklist

- [x ] commit message follows commit guidelines

##### Description of change

Add Angular Support Document and error solution
添加Angular支持文档

### CODE REVIEW

File: packages/site/docs/manual/FAQ/angular-support.en.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+However, if your application is upgraded from older angular version (for example, mine project is from angular 6), you may run into trouble.
+
+If you see `cannot read property 'webpackChunkAlgorithm'` error like following picture shows
+![image](https://user-images.githubusercontent.com/12276316/110507994-8e108e00-80ce-11eb-9f40-653f2181e44b.png)
+
+please refer to https://github.com/antvis/G6/issues/2691 for solution
+
+basically to make G6 work for angular:
+
+1. you need to have a .browserslistrc file under your application directory, same level with your package.json, `please be aware that browserslist is not working.`
```
Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.en.md:14]: `please be aware that browserslist is not working.`

->

**notice that the file browserslist won't work**.

Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.en.md:14]: you need to have a .browserslistrc file under your application directory, same level with your package.json

-> Create a .browserslistrc file under your application directory, same level as package.json

File: packages/site/docs/manual/FAQ/angular-support.en.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## This doc is only for Angular 9 and above and the solution is only tested on Angular 9 and above.
+
+There's no special settings or changings to use G6 in Angular if you just start a new Angular project from latest Angular CLI. Do it like you normal do when adding new dependency
+
+However, if your application is upgraded from older angular version (for example, mine project is from angular 6), you may run into trouble.
+
+If you see `cannot read property 'webpackChunkAlgorithm'` error like following picture shows
+![image](https://user-images.githubusercontent.com/12276316/110507994-8e108e00-80ce-11eb-9f40-653f2181e44b.png)
+
+please refer to https://github.com/antvis/G6/issues/2691 for solution
```
Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.en.md:10]: please refer to -> Please refer to

File: packages/site/docs/manual/FAQ/angular-support.en.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+## This doc is only for Angular 9 and above and the solution is only tested on Angular 9 and above.
+
+There's no special settings or changings to use G6 in Angular if you just start a new Angular project from latest Angular CLI. Do it like you normal do when adding new dependency
```
Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.en.md:3]: changings -> changes / differences

Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.en.md:3]: Do it like you normal do when adding new dependency

->

Follow the normal way when adding new dependencies.

File: packages/site/docs/manual/FAQ/angular-support.en.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+However, if your application is upgraded from older angular version (for example, mine project is from angular 6), you may run into trouble.
+
+If you see `cannot read property 'webpackChunkAlgorithm'` error like following picture shows
+![image](https://user-images.githubusercontent.com/12276316/110507994-8e108e00-80ce-11eb-9f40-653f2181e44b.png)
+
+please refer to https://github.com/antvis/G6/issues/2691 for solution
+
+basically to make G6 work for angular:
```
Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.en.md:12]: basically to make G6 work for angular:

-> 

To make G6 work for Angular:

File: packages/site/docs/manual/FAQ/angular-support.zh.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+## 本文档仅对于Angular 9以上版本适用
+如果你的项目是通过最新的Angular CLI创建,那么大概率G6可以即开即用,不会遇到什么问题,也不需要特殊设置.
```
Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.zh.md:2]: 最新的Angular CLI创建,那么大概率G6可以即开即用,不会遇到什么问题,也不需要特殊设置.

 -> 

最新的 Angular CLI 创建，那么大概率 G6 可以即开即用，不会遇到什么问题，也不需要特殊设置。

File: packages/site/docs/manual/FAQ/angular-support.zh.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+如果你的项目是从老的Angular版本升级而来(比如我的是从Angular 6逐步升级),那么你大概会遇到问题.
```
Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.zh.md:4]: 如果你的项目是从老的 Angular 版本升级而来（比如我的是从 Angular 6 逐步升级），那么你大概会遇到问题。

File: packages/site/docs/manual/FAQ/angular-support.zh.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+如果你遇到类似下图的 `cannot read property 'webpackChunkAlgorithm'` 错误
+![image](https://user-images.githubusercontent.com/12276316/110507994-8e108e00-80ce-11eb-9f40-653f2181e44b.png)
+
+请看 https://github.com/antvis/G6/issues/2691 解决
```
Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.zh.md:9]: 请看 https://github.com/antvis/G6/issues/2691 解决

->

请参考 https://github.com/antvis/G6/issues/2691 中的解决方案。

File: packages/site/docs/manual/FAQ/angular-support.zh.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+基本上要在Angular中使用G6需要满足以下两点:
```
Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.zh.md:11]: 在 Angular 中使用 G6，需要满足以下两点：

File: packages/site/docs/manual/FAQ/angular-support.zh.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+1. 项目根目录下(与package.json同目录) 需要有`.browserslistrc`文件, 请注意`browserslist`文件无效.
```
Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.zh.md:12]: 项目根目录下（与 package.json 同目录）需要有 `.browserslistrc` 文件，请注意 `browserslist` 文件无效。

File: packages/site/docs/manual/FAQ/angular-support.zh.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+2. **不**开启IE支持
```
Comment (Yanyan-Wang) [inline packages/site/docs/manual/FAQ/angular-support.zh.md:13]: **不**开启 IE 支持

**Review by Yanyan-Wang — APPROVED** [review]

**Comment by Yanyan-Wang** [comment]
Hi, it is appreciate for your PR! It would be helpful for Angular users! 

There are some minor problems about the spelling in English and punctuation in Chinese. It will be better if you could refine it.

Thanks again.

**Comment by wunaidage** [comment]
修改好了，谢谢

File: packages/site/docs/manual/FAQ/angular-support.en.md [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+
+If you see `cannot read property 'webpackChunkAlgorithm'` error like following picture shows
+![image](https://user-images.githubusercontent.com/12276316/110507994-8e108e00-80ce-11eb-9f40-653f2181e44b.png)
+
+Please refer to https://github.com/antvis/G6/issues/2691 for solution
+
+basically to make G6 work for angular:
+
+1. Create a .browserslistrc file under your application directory, same level as package.json. `please notice that browserslist won't working.`
+2. opt-out the IE support.
```
Comment (simplejason) [inline packages/site/docs/manual/FAQ/angular-support.en.md:15]: Angular still supports [IE 11](https://angular.io/guide/browser-support), does g6 support working under IE 11?

Comment (wunaidage) [reply packages/site/docs/manual/FAQ/angular-support.en.md:15]: I rememeber Angular has dropped IE 11 support since Angular 10.

Comment (simplejason) [reply packages/site/docs/manual/FAQ/angular-support.en.md:15]: In v11, it's true that Angular framework removes IE 9, 10, but IE 11 is not included(here is the [doc](https://angular.io/guide/browser-support)). It's ok for now, just a tiny nit :)

**Pull request merged**